.gitignore

```diff
@@ -48,6 +48,7 @@ docs/
 *.txt
 *.gz
 occurrence.RData
+/datasets/fornalab-formica-wp1-vegetation/data/raw
```

Comment:
Avoid pushing any raw data to Github, there are some unneccesairy lines in .gitignore